src/PubSubClient.h

```diff
@@ -83,7 +83,9 @@
 class PubSubClient {
 private:
    Client* _client;
-   uint8_t buffer[MQTT_MAX_PACKET_SIZE];
+   uint8_t * buffer;
+   uint16_t MQTT_MAX_PACKET_SIZE;
+   uint8_t _buffer[_MQTT_MAX_PACKET_SIZE];
```

Comment:
Minor, but `_buffer` is scarily close to `buffer`, makes it easy to accidentially misuse. Think this should have a more verbose/ugly name. Like `_staticInternalBuffer`.